e2e-tests/src/main/java/com/dnastack/ga4gh/dataconnect/adapter/DataConnectE2eTest.java

```diff
@@ -717,7 +822,7 @@ public void getTableData_should_returnDataAndDataModel() throws Exception {
     }
 
     @Test
-    public void getTables_should_require_searchInfo_scope() throws Exception {
+    public void getTables_should_require_searchInfo_scope() {
```

Comment:
Absolutely no need to put the `throws Exception` back, but I'll tell you why I tend to leave these on `@Test` methods (and only on `@Test` methods!) even when they are no longer needed.

The JUnit framework doesn't care if we throw checked vs. unchecked exceptions, and having `throws Exception` in place on a test method means one less thing that could go wrong in the future. What could go wrong? If someone introduces new code to the test that throws a checked exception, and they "handle" it with `catch (Exception e) {}` or even `catch (Exception e) { e.printStackTrace(); }` that's a new bug. If the test method already allows any exception to be thrown, the invitation to disaster isn't there. :)

But perhaps this was generating a warning in IntelliJ? That makes the decision a little harder, since lots of pointless warnings in the code tend to blind us to new and/or important warnings we might otherwise notice.